src/test/java/seedu/address/logic/commands/EventCommandTest.java

```diff
@@ -0,0 +1,294 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+import static seedu.address.testutil.EventBuilder.DEFAULT_DATE;
+import static seedu.address.testutil.EventBuilder.DEFAULT_INDEXES;
+import static seedu.address.testutil.EventBuilder.DEFAULT_NAME;
+
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.function.Predicate;
+
+import org.junit.jupiter.api.Test;
+
+import javafx.collections.ObservableList;
+import seedu.address.commons.core.GuiSettings;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.AddressBook;
+import seedu.address.model.EventBook;
+import seedu.address.model.Model;
+import seedu.address.model.ReadOnlyAddressBook;
+import seedu.address.model.ReadOnlyEventBook;
+import seedu.address.model.ReadOnlyUserPrefs;
+import seedu.address.model.event.Event;
+import seedu.address.model.person.Person;
+import seedu.address.model.util.SampleDataUtil;
+import seedu.address.testutil.EventBuilder;
+
+public class EventCommandTest {
+
+    @Test
+    public void constructor_nullParameters_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new EventCommand(null, null, null));
+
+        // test if any field is null
+        assertThrows(NullPointerException.class, () -> new EventCommand(null, DEFAULT_DATE, DEFAULT_INDEXES));
+        assertThrows(NullPointerException.class, () -> new EventCommand(DEFAULT_NAME, null, DEFAULT_INDEXES));
+        assertThrows(NullPointerException.class, () -> new EventCommand(DEFAULT_NAME, DEFAULT_DATE, null));
+    }
+
+    @Test
+    public void execute_eventAcceptedByModel_addSuccessful() throws Exception {
+        ModelStubAcceptingEventAdded modelStub = new ModelStubAcceptingEventAdded();
+        Event validEvent = new EventBuilder().buildWithNoAttendees();
+        CommandResult commandResult =
+                new EventCommand(DEFAULT_NAME, DEFAULT_DATE, DEFAULT_INDEXES).execute(modelStub);
+
+        assertEquals(String.format(EventCommand.MESSAGE_SUCCESS, Messages.formatEvent(validEvent)),
+                commandResult.getFeedbackToUser());
+        assertEquals(Arrays.asList(validEvent), modelStub.eventsAdded);
+    }
+
```

Comment:
Perhaps we can add a test case that adds an attendee to an event successful `execute_eventAcceptedByModel_addOneAttendee_addSuccessful()`. If I'm not wrong, the `DEFAULT_INDEXES` is an empty set.